app/controllers/todos_controller.rb

```diff
@@ -0,0 +1,56 @@
+class TodosController < ApplicationController
+  def index
+    @incomplete_todos = Todo.where(completed: false).order(due_date: :asc)
+    @complete_todos = Todo.where(completed: true).order(due_date: :asc)
```

Comment:
Someone else in our group also came up with this idea. I think it's such a smart and simple solution to split up and display the todos.

Reply:
Like Nitin said, we came up with a similar idea in our project. Another way to do it would be `Todo.order(:completed, :due_date`.

Reply:
Ah, yes I like the idea of ordering by two columns. That sounds like a much more elegant solution. Thanks for the suggestion!